corehq/apps/data_interfaces/forms.py

```diff
@@ -422,6 +445,26 @@ def __init__(self, domain, *args, **kwargs):
 
         self.custom_filters = settings.AVAILABLE_CUSTOM_RULE_CRITERIA.keys()
 
+    def user_locations(self):
+        if self.couch_user is None:
+            return []
+
+        if self.couch_user.is_domain_admin(self.domain):
```

Comment:
I am surprised that there isn't already a method to fetch these, have you checked if you could reuse anything?

Reply:
@sravfeyn I see this is the way it's checked in a bunch of other places. I haven't found another cleaner way yet.